README.md

```diff
@@ -61,7 +61,7 @@ and you need to have `jq` and [the 1password CLI][1password-cli] installed.
 For example, to create a git tag for the Rust 2.0.0 release you can use:
 
 ```
-./with-rust-key.sh gpg tag -u FA1BE5FE 2.0.0 stable
+./with-rust-key.sh gpg tag -u 108F66205EAEB0AAA8DD5E1C85AB96E6FA1BE5FE 2.0.0 stable
```

Comment:
The changes in the rest of the files are fine, but I'd like to keep the shorter ID here. There is no chance of a collision happening (the `gpg` command is run inside an ephemeral `GPGHOME` with just that key in it), and the shorter ID doesn't hide the command we're trying to run.

Reply:
I've reverted the change in the readme :+1: